### ISSUE TITLE
PDFUA-192: Content in columns incorrectly ordered

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-7#192]
Jira issue originally created by user @BPeboeck:

This describes a failure condition where a multicolumn document is tagged in a way that does not match the [author’s intent](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#authors-intent). Accessibility requires that [content](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#content) is tagged with a [logical content order](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#logical-content-order) that reflects the [author’s intent](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#authors-intent). In this example, the author intended columns but the [logical content order](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#logical-content-order) does not reflect columns.

**Comment by zakkinsey** [comment]
@BPeboeck assigned issue to self

**Comment by zakkinsey** [comment]
Comment created by @MarkusErle:

Accepted. Clean up necessary.

**Comment by zakkinsey** [comment]
Comment created by @pkopicar:

UA Identifier removed

**Comment by zakkinsey** [comment]
@BPeboeck added a link to #191: This issue relates to #191

**Comment by zakkinsey** [comment]
@BPeboeck added a link to #191: This issue relates to #191

**Comment by zakkinsey** [comment]
@BPeboeck removed a link to #191: This issue relates to #191

**Comment by zakkinsey** [comment]
@BPeboeck removed a link to #191: This issue relates to #191

**Comment by zakkinsey** [comment]
Comment created by @calanca2:

Put in Glossary page: 

 

The term “Logical Reading Order” refers to the sequential presentation of content in a PDF by screen readers and other assistive technology. Ideally, the screen reader will read the content in a manner that is logical and closely, if not exactly, matching the document's visual appearance.

**Comment by zakkinsey** [comment]
@calanca2 changed assignee from self to @PaulRayius

**Comment by zakkinsey** [comment]
@calanca2 unassigned issue from @PaulRayius

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

For metadata, I added Matterhorn 01-006 and WCAG 1.3.1 because, while it might be a little bit of a stretch, where the test procedures mention the reading order "matching the author's intent" I would also take that to imply that the tag type should also match the author's intent.  

Moving this along for review. 

(This is the same comment I added to #191.)

**Comment by zakkinsey** [comment]
Comment created by @willkil:

I do believe 01-006 and WCAG 1.3.1 are too much of a stretch for #191, #192, #235, and #236.

**Comment by zakkinsey** [comment]
Comment created by Iacobien Riezebosch `<iacobien@firmground.nl>`:

I am not sure about this example. It has two tests in the check. And check 1 should be false.

First question: what about the second check? There is no expected result for the second test.

Second question: in the example there is a problem with the content of the tags. Each tag contains a full line. I don't think this matches with a fail on test 1. The problem seems to be that there are no separe tags for colum 2. Agree?

Third: why is every line a paragraph? I think this example is confusing.

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

The file passes in Acrobat Preflight validation (as expected, probably), because, from a syntax perspective, it's ok.
